test/compute/2019-12-01/armcompute/zz_generated_availabilitysets_client.go

```diff
@@ -212,11 +212,11 @@ func (client *AvailabilitySetsClient) getHandleResponse(resp *http.Response) (Av
 	return result, nil
 }
 
-// List - Lists all availability sets in a resource group.
+// NewListPager - Lists all availability sets in a resource group.
 // If the operation fails it returns an *azcore.ResponseError type.
 // resourceGroupName - The name of the resource group.
 // options - AvailabilitySetsClientListOptions contains the optional parameters for the AvailabilitySetsClient.List method.
-func (client *AvailabilitySetsClient) List(resourceGroupName string, options *AvailabilitySetsClientListOptions) *runtime.Pager[AvailabilitySetsClientListResponse] {
+func (client *AvailabilitySetsClient) NewListPager(resourceGroupName string, options *AvailabilitySetsClientListOptions) *runtime.Pager[AvailabilitySetsClientListResponse] {
```

Comment:
@JeffreyRichter example of the pager API rename.

Reply:
I like it!